src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -281,6 +281,22 @@ def execution_environment(self, entry: SettingsEntry, config) -> PostProcessorRe
                         )
                     if hint:
                         exit_messages.append(ExitMessage(message=hint, prefix=ExitPrefix.HINT))
+
+            # check for /dev/mqueue/ when using podman because runner passes ipc=host
+            # https://github.com/ansible/ansible-navigator/issues/610
+            mqueue_path = "/dev/mqueue/"
+            if config.container_engine == "podman":
+                if not Path(mqueue_path).exists() or not Path(mqueue_path).is_dir():
+                    exit_msg = (
+                        "Execution environment support while using podman requires a"
+                        f" '{mqueue_path}' directory."
+                    )
+                    exit_messages.append(ExitMessage(message=exit_msg))
+                    hint = (
+                        "Try creating it with 'mkdir {mqueue_path}' or reference the documentation"
```

Comment:
Looks like this is missing an f-string prefix.
```suggestion
                        f"Try creating it with 'mkdir {mqueue_path}' or reference the documentation"
```
Is this not covered with tests?